doc/changelog.rst

```diff
@@ -38,6 +39,9 @@ Detailed Notes
   satisfied, the `Experiment` will attempt to wrap the underlying run command in a batch job using
   the object referenced at `Model.batch_settings` as the batch settings for the job. If the check
   is not satisfied, the `Model` is launched in the traditional manner as a job step. (PR245_)
+- The support for Ray was dropped, as its most recent versions caused problems when deployed through SmartSim.
+  We plan to release a separate add-on library to accomplish the same results. If
+  you are interested in getting the Ray launch functionality back in your workflow, please get in touch with us!
```

Comment:
Just following the pattern of the `changelog.rst`, should we link this PR here?

Reply:
Totally, I did not have the link until it was published :)